src/utils/index.js

```diff
@@ -110,6 +110,20 @@ const getErrorMessages = (error) => {
   return ['Unknown error.'];
 };
 
+const pricesEqual = (leftPrices, rightPrices) => {
```

Comment:
Can we rename these arguments to `initialPrices` and `currentPrices`? I don't love left and right. Same goes for all the left and right throughout the function

Reply:
Hrm. I wrote this in the mindset that it was a generic compare method. Like if I wrote a `strcmp` function, I wouldn't name the arguments after one particular use of it.

But since this is so tightly bound to its usage in this file (and isn't even exported), I'm down with renaming to make it more readable.

Reply:
I'm good with keeping it general. Left right feels weird. Maybe old new? I don't know. I trust your judgement